####-repo-rename.md

```diff
@@ -0,0 +1,150 @@
+# Plan to rename `Qiskit/qiskit-terra` repo to `Qiskit/qiskit`
+
+| **Status**        | **Proposed** |
+|:------------------|:---------------------------------------------|
+| **RFC #**         | ####                                         |
+| **Authors**       | [Luciano Bello](https://github.com/1ucian0), [Kevin Krsulich](https://github.com/kdk)  |
+| **Submitted**     | 2023-03-31                                   |
+| **Updated**       | YYYY-MM-DD                                   |
+
+
+## Summary
+In 2021, Qiskit began an evolution from a large monolithic install, which included by default the Aer simulator, the IBM Quantum provider, and large characterization and application libraries, into a leaner and more modular structure.
```

Comment:
Maybe it's worth putting a link to the blog post that announced that: https://research.ibm.com/blog/qiskit-application-modules (I never quite understood why we published that on the ibm research blog it should have been on the qiskit blog).